src/parser/Parser/Parser.hs

```diff
@@ -792,14 +792,18 @@ mutModifier :: EncParser Mutability
 mutModifier = (reserved "var" >> return Var)
           <|> (reserved "val" >> return Val)
 
+
 fieldDecl :: EncParser FieldDecl
-fieldDecl = do
+fieldDecl = do 
   fmeta <- buildMeta
   fmut  <- mutModifier
   fname <- Name <$> identifier
   colon
   ftype <- typ
-  returnWithEnd Field{fmeta, fmut, fname, ftype}
+  optional $ withLinebreaks $ reservedOp "="
+  fexpr <- optional expression
+  returnWithEnd Field{fmeta, fmut, fname, ftypr, fexpr}
```

Comment:
is this `returnWithEnd Field{fmeta, fmut, fname, ftypr, fexpr}` actually `returnWithEnd Field{fmeta, fmut, fname, ftype, fexpr}`  (misspelling of `ftypr` to `ftype`)